@xen-orchestra/log/logger.test.js

```diff
@@ -0,0 +1,154 @@
+'use strict'
+
+const { beforeEach, describe, it } = require('node:test')
+const { configure } = require('@xen-orchestra/log/configure')
+const { createCaptureTransport } = require('@xen-orchestra/log/capture')
+const { createLogger } = require('@xen-orchestra/log')
+const assert = require('node:assert/strict')
+
+describe('logger', () => {
+  const logger = createLogger('test-logger')
+
+  const logsTransportDefault = []
+  const transportTest = message => {
+    logsTransportDefault.push(message)
+  }
+
+  const logsTransportFatal = []
+  const transportFatal = message => {
+    logsTransportFatal.push(message)
+  }
+
+  beforeEach(() => {
+    configure(
+      createCaptureTransport([
+        {
+          level: 'fatal',
+          transport: transportFatal,
+        },
+        {
+          level: 'debug',
+          transport: transportTest,
+        },
+      ])
+    )
+
+    logsTransportDefault.length = 0
+    logsTransportFatal.length = 0
+  })
+
+  it('should log test with all its attributes', () => {
+    const expected1 = {
+      level: 20,
+      namespace: 'test-logger',
+      message: 'test 1 with all its attributes',
+      time: new Date(),
+    }
```

Comment:
Why are you creating this object? you are not using it? :thinking:

Reply:
The message is tested line 49 :+1: 
`    assert.equal(logsTransportDefault[0].message, expected1.message)`

Reply:
Yes but you don't need this object, you can simply create a `message` variable if that's all  you need.